tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
-    "target": "es6",
-    "strict": true,
+    "target": "es2018",
```

Comment:
Any idea what version of Node.js is required to run "es2018"?
Is there a reason why we should change it from "es6", which will work on earlier Node.js versions?

Reply:
Ok, this has been reverted. I don't have the option in the UI here, but this is "resolved".